Updated Providing_base_cases.ipynb to use google-genai

Shi-pra-19 opened the pull request:
Updated module google-generativeai to use google-genai
Refactored imports

Shi-pra-19 committed:
Updated Providing_base_cases.ipynb to use google-genai

Giom-V commented on examples/prompting/Providing_base_cases.ipynb:
```diff
@@ -6,7 +6,7 @@
         "id": "gy3KpnNx_Jl4"
```
<pre class="ql-syntax" spellcheck="false">Line #2.    You are an assistant that helps tourists around the world to plan their vacation. Your responsibilities are:
</pre><p>Can you add some indentation to improve readability? Like</p>

<pre class="ql-syntax" spellcheck="false">instructions = """
     You are an assistant that helps tourists around the world to plan their vacation. Your responsibilities are:
     1. Helping book the hotel.
     2. Recommending restaurants.
     3. Warning about potential dangers.

     If other request is asked return "I cannot help with this request."
"""
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/643/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Giom-V replied:
<pre class="ql-syntax" spellcheck="false">Line #1.    print("ON TOPIC:", on_topic_response.text)
</pre><p>Can you use <code>display(Markdown())</code> to improve the rendering of the markdown?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/643/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Giom-V replied:
<p>Can you add indentation here as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/643/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Giom-V replied:
<pre class="ql-syntax" spellcheck="false">Line #1.    print('''## Specified genre:
</pre><p>Can you use <code>display(Markdown())</code> to improve the rendering of the markdown?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/643/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Giom-V commented:
Thanks for the submission @Shi-pra-19! It's the first one if I'm not mistaken 🥳 

I just added a couple of minor comments to make the notebook easier to understand.

Can you also check the format failures. It is likely because you haven't run the formatting script.

Thanks again!

Shi-pra-19 committed:
Merge branch 'google-gemini:main' into base-cases

Shi-pra-19 committed:
Used Markdown display for better formatted responses and improved indentation.

Shi-pra-19 commented:
@Giom-V Thank you for the review! This one is my second submission—my first one was #642. 🥳

I’ve added indentation and run the formatting script. Let me know if there are any more improvements needed.

Also, please take a look at #642 as well—I fixed the indentation there too.

Shi-pra-19 committed:
Merge branch 'google-gemini:main' into base-cases

Shi-pra-19 committed:
nbfmt

andycandy commented on examples/prompting/Providing_base_cases.ipynb:
```diff
@@ -11,7 +11,7 @@
     },
```
<p>Could you update the model list so we can get this merged quickly?</p>

<pre class="ql-syntax" spellcheck="false">MODEL_ID="gemini-2.5-flash-preview-05-20" # @param ["gemini-2.5-flash-preview-05-20", "gemini-2.5-pro-preview-05-06"] {"allow-input":true, isTemplate: true}
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/643/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

andycandy replied:
<p>Could you wrap this line up so its easier to read?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/643/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Shi-pra-19 committed:
Merge branch 'google-gemini:main' into base-cases

Shi-pra-19 committed:
updated model list

Shi-pra-19 committed:
nbfmt

Shi-pra-19 commented:
Fixed it. Let me know if I missed anything.

andycandy commented:
> Fixed it. Let me know if I missed anything.

Looks great overall! Just one small thing, could you avoid bumping the `@google/genai` package to 1.7.0? Since the current code works with versions as low as 1.0.0, keeping it more broadly compatible would be ideal. Once that's updated, we can get this merged right away :)

Shi-pra-19 committed:
Updated google-genai version

Shi-pra-19 committed:
Merge branch 'google-gemini:main' into base-cases

Shi-pra-19 committed:
nbfmt

Shi-pra-19 committed:
Update model ID
